components/size_aware_image.jsx

```diff
@@ -139,33 +215,21 @@ export default class SizeAwareImage extends React.PureComponent {
                 </div>
             );
         }
-        Reflect.deleteProperty(props, 'showLoader');
-        Reflect.deleteProperty(props, 'onImageLoaded');
-        Reflect.deleteProperty(props, 'onImageLoadFail');
 
         //The css hack here for loading images in the background can be removed after IE11 is dropped in 5.16v
         //We can go back to https://github.com/mattermost/mattermost-webapp/pull/2924/files
-        if (!this.state.loaded && dimensions && dimensions.width) {
+        if (!this.state.loaded && this.dimensionsAvailable(dimensions)) {
             imageStyleChangesOnLoad = {position: 'absolute', top: 0, height: 1, width: 1, visibility: 'hidden', overflow: 'hidden'};
```

Comment:
Can we move this to a class somewhere?

Reply:
Not sure how best to handle this for a workaround - based on the code comment above this line -, I would love for @sudheerDev to chime in as he has more context about this

Reply:
@devinbinnie, @sudheerDev I will be waiting for your feedback on this to complete the requested changes and push my local updates

Reply:
Well I know IE11 is no longer supported by Mattermost, so it should be safe to remove altogether, @sudheerDev does that make sense?

Reply:
In case it is gonna be removed altogether, is it better to do so in a separate PR? (so it is easier to isolate the change in case there are any regression issues because of it)

Reply:
Either that or at least a separate commit.

Reply:
A separate commit sounds good. I can do so once Sudheer confirms or provides his feedback.

Reply:
@abdusabri @devinbinnie Yes, we can remove that as we don't need to support for IE11. IMO it should be a separate PR.

Reply:
>  it is easier to isolate the change in case there are any regression issues because of it

I agree with this. i think we should isolate that change. We already have a lot in this PR already. Better not to make more changes on top of this. It is easier for git blame to track and understand why a change was made as well if it is in separate PR.

Reply:
Ok, then let's do this in a separate PR.

Please feel free to create a ticket and assign it to me (@devinbinnie or @sudheerDev)